app/assets/stylesheets/common/base/_topic-list.scss

```diff
@@ -329,6 +329,7 @@
     .badge-posts {
       font-weight: 700;
       color: inherit;
+      display: block;
```

Comment:
Just checking: this will be a no-op for the old topic list, because the old button already has `display: block`?

Reply:
double checked and it's `inline-block`. it usually doesn't matter but in case there is something in the outlet next to it, I'm changing it to match non-glimmer css